cluster/senza-definition.yaml

```diff
@@ -47,8 +51,7 @@ SenzaComponents:
           LoadBalancerPort: 443
       ConnectionSettings:
         IdleTimeout: 300
-      VersionDomain: "{{SenzaInfo.StackName}}-{{SenzaInfo.StackVersion}}.{{Arguments.HostedZone}}"
-      MainDomain: "{{SenzaInfo.StackName}}.{{Arguments.HostedZone}}"
+      VersionDomain: "{{ Arguments.APIServerHostName }}"
```

Comment:
I think we have to define `MainDomain` here too, otherwise Senza will ask the user what domain to choose in case of multiple hosted zones!